UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjective.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace StationObjectives
+{
+	public abstract class StationObjective : ScriptableObject
+	{
+		/// <summary>
+		/// The description of the objective which is shown to players.
+		/// </summary>
+		[SerializeField]
+		public string description;
```

Comment:
make it private with a public getter or remove the decorator since publics are serialzied by default (please choose former)